db-acra-migration/python/requirements/base.txt

```diff
@@ -0,0 +1,3 @@
+SQLAlchemy==1.4.15
+pythemis
+acrawriter
```

Comment:
do we really need here acrawriter? do we use acrastructs anywhere? if so, let's don't use )) due to they useful only for e2e encryption that we don't have here